clickhouse_plantuml/table.py

```diff
@@ -63,6 +63,15 @@ def __init__(
         self.sampling_key = sampling_key
         self.columns = []  # type: List[Column]
 
+    def parse_additional_dependencies(self):
```

Comment:
It looks to me, as it should be used in the `Tables.merge_matviews` now.

Since it's related to MatViews only, `parse_matview_dependency` sounds a little bit better.

And I feel like you forgot about `.inner.table_name` part https://github.com/Felixoid/clickhouse-plantuml/blob/774cfafee30020bf28eeb8ab8c07d6182e32ec22/clickhouse_plantuml/tables.py#L160-L167